src/compiler/scanner.ts

```diff
@@ -1031,6 +1032,8 @@ export function createScanner(languageVersion: ScriptTarget,
         scanRange,
     };
 
+    /* eslint-disable no-var */

```

Comment:
Drive-by - was this maybe supposed to be `eslint-enable`?

Reply:
Uh oh, sure was!